app/build.gradle

```diff
@@ -92,13 +92,18 @@ android {
             if (project.hasProperty('crashlyticsBetaBuild')) {
                 signingConfig signingConfigs.crashlyticsBeta
             }
+            //AppSee key
+            manifestPlaceholders = [appSeekey:"f05dbdbcd6524a7089bdcfbac5c9be11"]
```

Comment:
interesting, this is the first time I've seen this property

Reply:
I found it when I was doing some research on how to handle it.